components/http_foundation/introduction.rst

```diff
@@ -344,7 +344,37 @@ attribute::
 The
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::setCookie`
 method takes an instance of
-:class:`Symfony\\Component\\HttpFoundation\\Cookie` as an argument.
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` as an argument. This class
+takes eight arguments in its constructor:
+
+``$name``
+    **type**: ``string`` **default**: none (this argument is mandatory)
```

Comment:
There always needs to be a blank line after the first line of the explanation (otherwise we would have only one paragraph for each explanation).